apps/condo/domains/news/components/NewsForm/CreateNewsForm.tsx

```diff
@@ -29,16 +29,12 @@ export const CreateNewsActionBar: React.FC<{ handleSave: () => void, isLoading:
     const { handleSave, isLoading } = props
 
     return (
-        <ActionBar
-            actions={[
-                <Button
-                    key='submit'
-                    type='primary'
-                    children={ShareButtonMessage}
-                    onClick={handleSave}
-                    disabled={isLoading}
-                />,
-            ]}
+        <Button
```

Comment:
Looks very strange, function is called CreateNewsActionBar and we it returns just a button

Reply:
And how I may rename this function?
Reworking a function to match its current name greatly complicates the logic.

Reply:
I can help with naming, but I dont quite understand what you are trying to achieve here – why do you need to change CreateNewsActionBar ?